src/usr/local/www/services_captiveportal.php

```diff
@@ -633,6 +635,13 @@ function build_authserver_list() {
 	$pconfig['blockedmacsurl']
 ))->setHelp('Blocked MAC addresses will be redirected to this URL when attempting access.');
 
+$section->addInput(new Form_Checkbox(
+	'preservedb',
+	'Preserve users database',
+	'Preserve connected users across reboot',
+	$pconfig['preservedb']
+))->setHelp('If enabled, connected users won\'t be disconnected during a pfSense reboot.');
```

Comment:
Please use double quotes in the string and remove the escape from `win\'t`. It will make translation process easier